src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Private-DNS-Generic.json

```
@@ -0,0 +1,143 @@
+{
+    "name": "Deploy-Private-DNS-Generic",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "mode": "All",
+        "displayName": "Deploy-Private-DNS-Generic",
+        "description": "Configure private DNS zone group to override the DNS resolution for PaaS services private endpoint",
+        "metadata": {
+            "version": "1.0.0",
+            "category": "Networking",
+            "source": "https://github.com/Azure/Enterprise-Scale/",
+            "alzCloudEnvironments": [
+                "AzureCloud"
```

```suggestion
                "AzureCloud",
        	"AzureChinaCloud",
       		"AzureUSGovernment"
```
This policy will work in any cloud that private endpoints are supported, which is commercial, gov and China 👍